### PR TITLE
use requests library to fix #7

### LINKED ISSUE #7: SendInBlue SSL certificate not verified
The commit 1b69eb9d5d7edf421da3ee02b66d21233618719e in issue #3 disabled SSL cert validation by the API. I think that should be reverted and any SSL problems corrected by making sure that the certificate can be properly verified.

### PR DESCRIPTION
**Pull request by alanhamlett**
Only for V2.0 not V1.0.

### CODE REVIEW

File: V2.0/mailin.py [hunk 1]
```diff
@@ -20,15 +18,16 @@ def do_request(self,resource,method,indata):
...
-    socket.setdefaulttimeout(self.timeout)
 
-    h = httplib2.Http(".cache", disable_ssl_certificate_validation=True)
-    content_type = "application/json"
-    r,c = h.request(url,method,body=indata,headers={'api-key':self.api_key, 'content-type':content_type})
-    charenc = chardet.detect(c)
-    return json.loads(c.decode(charenc['encoding']))  
+    content_type = 'application/json'
+    headers = {
+        'api-key': self.api_key,
```
Comment (raftelti) [inline V2.0/mailin.py]: This part of the code uses 4-space indentation while the rest uses 2-space indentation.

Comment (alanhamlett) [inline V2.0/mailin.py]: Changing it to 2 spaces now. My editor defaults to the Python PEP8 standard 4 spaces:
https://www.python.org/dev/peps/pep-0008/#indentation

**Comment by alanhamlett** [comment]
``

**Comment by ekta-slit** [comment]
Hi @alanhamlett , we appreciate your suggestions!
Thank you

**Pull request merged**